src/pybind/mgr/smb/handler.py

```diff
@@ -1173,3 +1173,11 @@ def _cephx_data_entity(cluster_id: str) -> str:
     use for data access.
     """
     return f'client.smb.fs.cluster.{cluster_id}'
+
+
+def rand_name(prefix: str, max_len: int = 18, suffix_len: int = 8) -> str:
+    trunc = prefix[: (max_len - suffix_len)]
```

Comment:
it's probably unnecessary, but do we want to validate here that `max_len` is greater than `suffix_len`

Reply:
Next time I touch that function I'll add that